edgexfoundry/edgex-go
# feat!: consume common config

BREAKING CHANGE: removes the common config information for core services

closes https://github.com/edgexfoundry/edgex-go/issues/4263


**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I have added unit tests for the new feature or bug fix (if not, why?) - bootstrap load for common config
  <link to docs PR>

## Testing Instructions
1. run `make docker` to build the docker images
2. In compose builder run `make run no-secty dev` for non-secure mode testing or `make run dev` for secure mode testing.
3. ensure that all the services only have common config sections loaded into consul
4. ensure that all services - core data, core-metadata, etc start up properly

## New Dependency Instructions (If applicable)
